elytron/ELY-1778-enhanced-audit-logging-feature-parity-with-legacy.adoc

```diff
@@ -0,0 +1,125 @@
+= ELY- Enhanced Audit Logging - Feature Parity with Legacy Audit Logging
+:author:            Justin Michael Cook
+:email:             jucook@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Currently, Audit Logging in Elytron does not have the same capabilities as Legacy Security. This task is to enhance Elytron Audit Logging to have the same capabilities as Legacy Security Audit Logging through the addition of an @Audit annotation and additional audit events.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/ELY-1778[ELY-1778]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1230[EAP7-1230]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:jkasik@redhat.com[Jan Kašík]
+
+=== Testing By
+
+[X] Engineering
+
+[ ] QE
```

Comment:
@justinmcook Can you please switch this to QE (as discussed before)?

Reply:
Fixed